### PR TITLE
release 0.5.0

### PR DESCRIPTION
**Pull request by krasi-georgiev**
cc @codesome 

Signed-off-by: Krasi Georgiev <kgeorgie@redhat.com>

### CODE REVIEW

**Comment by hoffie** [comment]
Maybe include #531 in the changelog as a bugfix for a performance regression in the previous version?

**Comment by krasi-georgiev** [comment]
@hoffie 	added, thanks

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,9 @@
     - Added to remove some code duplication.
  - [ENHANCEMENT] When closing the db any running compaction will be cancelled so it doesn't block.
    - `NewLeveledCompactor` takes a context.
- - [CHANGE] `prometheus_tsdb_storage_blocks_bytes_total` is now `prometheus_tsdb_storage_blocks_bytes`
+ - [CHANGE] `prometheus_tsdb_storage_blocks_bytes_total` is now `prometheus_tsdb_storage_blocks_bytes`.
+ - [BUGFIX] Improved Merge performance.
```
Comment (bwplotka) [inline CHANGELOG.md]: Is that really a bugifx?

Comment (brian-brazil) [reply CHANGELOG.md]: Yes, it fixed a regression from previous performance improvements.

Comment (codesome) [reply CHANGELOG.md]: Should we name it "Improved Postings Merge performance"? Given that we have query merge now, this can be confusing.

Comment (krasi-georgiev) [reply CHANGELOG.md]: updated.

**Pull request merged**